scheduler/feasible.go

```diff
@@ -1164,3 +1167,46 @@ func checkAttributeConstraint(ctx Context, operand string, lVal, rVal *psstructs
 	}
 
 }
+
+type VerConstraints interface {
```

Comment:
Do these (VerConstraints, VerConstraintParser) have to be exported?

Reply:
Neither has to be, but VerConstraints is used in exported methods, so I'd prefer to export that one for clarity and documentation.

VerConstraintParser is never exposed, so I made it private.